Name: Hide NASCAR App
Version: 1.4.0-1
Author: dBsooner, noir, cryptk
Description: This will hide the Nascar application in the App Launcher.

--- .orig/usr/palm/applications/com.handson.app.nascar/appinfo.json
+++ /usr/palm/applications/com.handson.app.nascar/appinfo.json
@@ -7,6 +7,7 @@
 	"splashicon": "icon-256x256.png",
 	"version": "1.24.0",
 	"vendor": "Hands-On Mobile",
-	"vendorurl": "http://www.handson.com" 
+	"vendorurl": "http://www.handson.com",
+	"visible": false
 }
 
